vim-impl/src/test/java/org/openbaton/vim_impl/vim/test/VimTestSuiteClass.java

```diff
@@ -103,7 +100,7 @@
 
   //    @InjectMocks
   //@Qualifier("OpenstackVim")
```

Comment:
as you are cleaning up, I suggest to remove also old comments not applicable anymore